pkg/kubelet/kubelet.go

```diff
@@ -1962,7 +1962,7 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 	if err := result.Error(); err != nil {
 		// Do not return error if the only failures were pods in backoff
 		for _, r := range result.SyncResults {
-			if r.Error != kubecontainer.ErrCrashLoopBackOff && r.Error != images.ErrImagePullBackOff {
+			if !errors.Is(r.Error, kubecontainer.ErrCrashLoopBackOff) && !errors.Is(r.Error, images.ErrImagePullBackOff) {
```

Comment:
Thank you for your PR! Can you fix similar errors in other places in pkg/kubelet if any?

Reply:
I have submitted a new commit, please review, thanks.